### DISCUSSION TITLE
UI design

### DISCUSSION THREAD

**Discussion by yashbrid03** [stephin007/Cowin-Vaccine-Availablity-Checker#7] [General]
What are the modules in the web-app so that we can start to design the UI

**Comment by stephin007** [comment]
Hey @yashbrid03 could you be more specific please?

**Reply by yashbrid03** [reply]
@stephin007 like which information we have to showcase in our Web-App? eg: No. of vaccines available, No. of vaccination centers like this

**Reply by stephin007** [reply]
This is the current UI: https://cowin-vaccine-availablity-checker-git-ui-fixes-stephin007.vercel.app/

Check this and comment what can be improved in this. @Justinnn07  is working on this, he will  do the necessary changes.

FYI: @yashbrid03 we are using Material UI.

**Reply by yashbrid03** [reply]
I think we should change the background image and replace it with solid color. So keeping dark theme in mind I have a color palette. have a look at this palette : **[https://colorhunt.co/palette/125503](https://colorhunt.co/palette/125503)**. What do you guys think? @Justinnn07 @stephin007 . If you guys have any other palette in mind, can suggest

**Reply by stephin007** [reply]
Looks good @yashbrid03 ⭐ 
can you please also suggest which color goes where?

@Justinnn07 does this enhancement looks doable? let me know your thoughts... if you some other ideas?

**Reply by Justinnn07** [reply]
> I think we should change the background image and replace it with solid color. So keeping dark theme in mind I have a color palette. have a look at this palette : **https://colorhunt.co/palette/125503**. What do you guys think? @Justinnn07 @stephin007 . If you guys have any other palette in mind, can suggest

cool

**Reply by Justinnn07** [reply]
> Looks good @yashbrid03 ⭐
> can you please also suggest which color goes where?
> 
> @Justinnn07 does this enhancement looks doable? let me know your thoughts... if you some other ideas?

- Am currently working with the changes you proposed in the PR!

**Reply by yashbrid03** [reply]
> Looks good @yashbrid03 ⭐
> can you please also suggest which color goes where?
> 
> @Justinnn07 does this enhancement looks doable? let me know your thoughts... if you some other ideas?

Yeah sure. Just give me some time and I'll clear the color arrangement

**Reply by stephin007** [reply]
Yeah i k @Justinnn07 , after that , you can do the color changes in the same PR. as suggested by @yashbrid03 
Can you do that?

@yashbrid03 can you please tell which color should go where?

**Reply by Justinnn07** [reply]
> Yeah i k @Justinnn07 , after that , you can do the color changes in the same PR. as suggested by @yashbrid03
> Can you do that?
> 
> @yashbrid03 can you please tell which color should go where?

did!

**Reply by stephin007** [reply]
@Justinnn07 for now, you add the colors which u feel like will look good then lets do the changes.

**Reply by Justinnn07** [reply]
> @Justinnn07 for now, you add the colors which u feel like will look good then lets do the changes.

cool

**Comment by Justinnn07** [comment]
@yashbrid03 it would be better if you make a wireframe on either draw.io or figma 

thanks

**Reply by stephin007** [reply]
@yashbrid03 this would be more effective! 
a visual look of what is to be done!

**Reply by yashbrid03** [reply]
> @yashbrid03 it would be better if you make a wireframe on either draw.io or figma
> 
> thanks

sure buddy

**Comment by yashbrid03** [comment]
I have made some logo for this project. If we want, we can use these logos. **[logos](https://www.figma.com/file/zPgZv1ajxXuApn5g0HFbbA/Logos?node-id=0%3A1)**

**Reply by stephin007** [reply]
Noted this, but before adding any logos
Can you please tell which color should go where in the UI

**Comment by stephin007** [comment]
@Justinnn07 you can check this link to get an idea about how to use colors to achieve dark mode! we need

**Reply by stephin007** [reply]
https://material.io/design/color/dark-theme.html

**Reply by Justinnn07** [reply]
> https://material.io/design/color/dark-theme.html

yeah, we can toggle dark theme!

**Comment by Justinnn07** [comment]
@yashbrid03 bro wireframe?

**Reply by stephin007** [reply]
he has left the team! @Justinnn07

**Comment by maxpainnogain1** [comment]
Hello Everyone , I have created this UI please let me know your reviews 

https://www.canva.com/design/DAEdz0Vwc8g/4YRVuiZYyqNWZisvlJhygw/edit 



![bacgroundimg](https://user-images.githubusercontent.com/71087810/117496553-d5af7b00-af94-11eb-84bb-913a1f386811.png)

<img width="375" alt="003" src="https://user-images.githubusercontent.com/71087810/117496136-3d18fb00-af94-11eb-876a-d3acc96aaa75.png">

**Reply by stephin007** [reply]
Wow Man, looks good.
simple yet classy

@Justinnn07 @vanshsehgal2001 lets discuss this and start working on it!

First lets begin with the home page and then shift to the rest of the pages.

**Reply by vanshsehgal2001** [reply]
Woah!!!

**Reply by vanshsehgal2001** [reply]
Yeah I talked to Justin and he said... Will do in tomorrow's meet

**Reply by Justinnn07** [reply]
> Hello Everyone , I have created this UI please let me know your reviews
> 
> https://www.canva.com/design/DAEdz0Vwc8g/4YRVuiZYyqNWZisvlJhygw/edit
> 
> ![bacgroundimg](https://user-images.githubusercontent.com/71087810/117496553-d5af7b00-af94-11eb-84bb-913a1f386811.png)
> 
> <img alt="003" width="375" src="https://user-images.githubusercontent.com/71087810/117496136-3d18fb00-af94-11eb-876a-d3acc96aaa75.png">

💯

**Reply by stephin007** [reply]
@teradaian , we are adding a home page for the project(as shown above^^), so adding the light ~ dark mode switch might need to be replaced to the home page, will it be lot of work?

you can implement the one you are doing right now, but just gave you a headsup! Thanks
